components/templates/src/global_fns/mod.rs

```diff
@@ -149,21 +149,37 @@ impl TeraFn for GetUrl {
             }
         } else {
             // anything else
-            let mut permalink = self.config.make_permalink(&path);
+            let mut pathbuf = PathBuf::new();
```

Comment:
Using PathBuf for urls won't work in Windows.

Reply:
Ack, that makes sense. I'll find another way.